frontend/src/scenes/settings/environment/ReplayTriggers.tsx

```diff
@@ -385,107 +398,27 @@ function Sampling(): JSX.Element {
-                            },
-                            {
-                                label: '55%',
-                                value: '0.55',
-                            },
-                            {
-                                label: '50%',
-                                value: '0.50',
-                            },
-                            {
-                                label: '45%',
-                                value: '0.45',
-                            },
-                            {
-                                label: '40%',
-                                value: '0.40',
-                            },
-                            {
-                                label: '35%',
-                                value: '0.35',
-                            },
-                            {
-                                label: '30%',
-                                value: '0.30',
-                            },
-                            {
-                                label: '25%',
-                                value: '0.25',
-                            },
-                            {
-                                label: '20%',
-                                value: '0.20',
-                            },
-                            {
-                                label: '15%',
-                                value: '0.15',
-                            },
-                            {
-                                label: '10%',
-                                value: '0.10',
-                            },
-                            {
-                                label: '5%',
-                                value: '0.05',
-                            },
-                            {
-                                label: '1%',
-                                value: '0.01',
-                            },
-                            {
-                                label: '0% (replay disabled)',
-                                value: '0.00',
-                            },
-                        ]}
-                        value={
-                            typeof currentTeam?.session_recording_sample_rate === 'string'
-                                ? currentTeam?.session_recording_sample_rate
-                                : '1.00'
-                        }
-                    />
+                    <div className="flex flex-row gap-x-2">
+                        <LemonInput
+                            type="number"
+                            className="[&>input::-webkit-inner-spin-button]:appearance-none"
+                            onChange={(value) => setValue(value)}
+                            min={0}
+                            max={100}
+                            suffix={<>%</>}
+                            value={value}
+                            onPressEnter={updateSampling}
+                            data-attr="sampling"
```

Comment:
```suggestion
                            data-attr="sampling-setting-input"
```

nit pick to avoid risk of collision elsewhere

Reply:
yeah more descriptive attribute sounds better, I'lll fit it in before merging 🙏